DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: r.releases.utils
 Title: Utilities for An R Universe of Package Releases
 Description: Utilities for an R universe of package releases.
-Version: 0.0.5
+Version: 0.0.6
```

Comment:
I added a commit to bump to the next prd version because this check is really important.